.gitlab/check_gem_presence.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+
+require 'bundler/inline'
+
+gemfile { gem 'gems', source: 'https://rubygems.org' }
+
+require 'rubygems'
+require 'gems'
+
+version = ARGV[0].chomp
+version = version.delete_prefix('v') if version.start_with?('v')
+
+candidate = Gem::Version.new(version)
+
+retry_count = 0
+max_retries = 60
+interval = 60
```

Comment:
Do we need wait a whole minute before retrying?
I ask in case `.gitlab/check_gem_presence.rb` is ever something a human is actively waiting on. If so, I suggest reducing the interval to something like 10 seconds (and increasing `max_retries` to match).

Reply:
We can change this frequency in the future.